wdl/transforms/biscayne/src/main/scala/wdl/transforms/biscayne/linking/expression/files/BiscayneFileEvaluators.scala

```diff
@@ -17,6 +28,10 @@ object BiscayneFileEvaluators {
 
   implicit val sepFunctionEvaluator: FileEvaluator[Sep] = twoParameterFunctionPassthroughFileEvaluator[Sep]
   implicit val suffixFunctionEvaluator: FileEvaluator[Suffix] = twoParameterFunctionPassthroughFileEvaluator[Suffix]
+  implicit val quoteFunctionEvaluator: FileEvaluator[Quote] =
+    EngineFunctionEvaluators.singleParameterPassthroughFileEvaluator
+  implicit val sQuoteFunctionEvaluator: FileEvaluator[SQuote] =
+    EngineFunctionEvaluators.singleParameterPassthroughFileEvaluator
```

Comment:
Nitpick for consistency - can you update the import to handle `singleParameterPassthroughFileEvaluator` the same way as `twoParameterFunctionPassthroughFileEvaluator` so they can be referenced the same way here?